Sources/Tracker/Entities/ClickStreamHealthConfigurations.swift

```diff
@@ -23,6 +23,18 @@ public struct ClickstreamHealthConfigurations {
     /// If you want to use a proto message name the you can directly call like this:- User.protoMessageName or CardEvent.protoMessageName
     private(set) var dropRateEventName: String
     
+    var verbosityLevel: String?
+    
+    static var logVerbose: Bool {
+        Clickstream.healthTrackingConfigs?.verbosityLevel?.lowercased() == "maximum"
+    }
+    
+    /// Returns an instance of ClickStreamEventClassification by decoding the json string.
+    /// - Parameter json: String that needs to be decoded.
+    static func getInstance(from json: String) -> ClickstreamHealthConfigurations? {
```

Comment:
JSONString is error prone, please revert back and use ClickstreamHealthConfigurations instance